pkg/state/filter_test.go

```diff
@@ -307,4 +307,66 @@ routes:
 			Expect(returnedState).To(MatchYAML(filteredState))
 		})
 	})
+	Context("when there is a linux bridge without 'bridge' options because is down", func() {
+		BeforeEach(func() {
+			state = nmstate.NewState(`
+interfaces:
+- name: eth1
+  state: up
+  type: ethernet
+- name: br1
+  type: linux-bridge
+  state: down
+  ipv4:
+    enabled: false
+  ipv6:
+    enabled: false
+  lldp:
+    enabled: false
+  mac-address: A2:EE:84:7B:42:4C
+  mtu: 1500
+routes:
```

Comment:
Please drop everything that is not necessary for this test. We do not care about routes, do we?